docs/database development process/overview.rst

```diff
@@ -0,0 +1,31 @@
+Overview
+=========
+
+SymbiFlow/prjxray/fuzzers/
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Fuzzers are things that generate a design, feed it to Vivado, and look at the resulting bitstream to make some conclusion.
+This is how the contents of the database are generated.
+
+The general idea behind fuzzers is to pick some element in the device (say a block RAM or IOB) to target.
+If you picked the IOB (no one is working on that yet), you'd write a design that is implemented in a specific IOB.
+Then you'd create a program that creates variations of the design (called specimens) that vary the design parameters, for example, changing the configuration of a single pin.
+
+A lot of this program is TCL that runs inside Vivado to change the design parameters, because it is a bit faster to load in one Verilog model and use TCL to replicate it with varying inputs instead of having different models and loading them individually.
+
+By looking at all the resulting specimens, you can correlate which bits in which frame correspond to a particular choice in the design.
+
+Looking at the implemented design in Vivado with "Show Routing Resources" turned on is quite helpful in understanding what all choices exist.
+
+SymbiFlow/symbiflow-arch-defs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This is where we describe the logical components in a device to VPR.
+VPR stands for place and route software.
```

Comment:
`VtR` == [Verilog to Routing](https://verilogtorouting.org)
`VPR` == **V**tR **P**lace and **R**oute

VtR also has a synthesis tool called `ODIN-II` -- we are using Yosys instead of that.